sycl/CMakeLists.txt

```diff
@@ -39,6 +39,8 @@ if(SYCL_ENABLE_WERROR)
   endif()
 endif()
 
+add_definitions(-g)
+
```

Comment:
```suggestion
```

Reply:
Thanks, I thought I reverted this change before making commit...apparently no

Reply:
Removed in ef3f16815e091a4d7a8d9d31583d9030e2ce3f21